docs/05-technical-reference/00-custom-resources/ac-01-application.md

```diff
@@ -2,7 +2,7 @@
 title: Application
 ---
 
-The `applications.applicationconnector.kyma-project.io` CustomResourceDefinition (CRD) is a detailed description of the kind of data and the format used to register an Application in Kyma. The `Application` custom resource (CR) defines the APIs that the Application offers. After creating a new custom resource for a given Application, the Application is mapped to appropriate ServiceClasses in the Service Catalog. To get the up-to-date CRD and show the output in the YAML format, run this command:
```

Comment:
I removed that 'cause it references Service Catalog. Should we replace it with anything or is it all good?

Reply:
That's ok.